orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroup.groovy

```diff
@@ -58,6 +58,9 @@ class TargetServerGroup {
     } else if (loc.type == Location.Type.ZONE) {
       op.zone = loc.value
       op.zones = [loc.value]
+    } else if (loc.type == Location.Type.NAMESPACE) {
+      op.namespace = loc.value
```

Comment:
Would really like to cleanup this pattern where we include the singular and plural of each location. Probably not the right time now though (as part of this PR, that is).